inlang/source-code/ide-extension/src/commands/extractMessage.ts

```diff
@@ -121,7 +121,6 @@ export const extractMessageCommand = {
 
 		telemetry.capture({
 			event: "IDE-EXTENSION command executed",
-			properties: { name: "extract message" },
```

Comment:
removed the redundant "name" property. the name of an event is the event name itself